main.tf

```
@@ -4,4 +4,42 @@
 # TEMPLATE:
 # TEMPLATE: When main.tf becomes unwieldy, consider submodules (https://www.terraform.io/docs/language/modules/develop/structure.html) 
 # TEMPLATE: and dependency inversion (https://www.terraform.io/docs/language/modules/develop/composition.html).
+# TEMPLATE:
+
+# TEMPLATE: Replace sample provider described below with your own.
+terraform {
+  required_version = ">= 1.3"
```

We can provide provider meta templating here:

```suggestion
  required_version = ">= 1.3"
  provider_meta "equinix" {
    module_name = "template" # TEMPLATE: Replace the module name with your own.
  }
```

---

@ocobleseqx does this seem like the format we would want to encourage? (plain name)

---

Thanks! I have added this into my latest update 2ecad88dc122f3fa8e342f5d0fa079e56f310112